app/views/users/extlogout.html.erb

```diff
@@ -1,19 +1 @@
-<% content_for(:title, _("Logout")) %>
-
-<div id="login" class="container">
-  <div class="row">
-    <div class="col-sm-12">
-      <div id="brand">
-        <%= image_tag("login_logo.png") %>
-      </div><!--/#brand-->
-    </div>
-    <div class="col-sm-7 col-md-6 col-lg-5 login">
-      <div class="form-group">
-        <div class="col-xs-offset-8 col-xs-4 submit">
-          <%= link_to(_("Click to log in again."), extlogin_users_path, :class => "btn btn-primary") %>
-        </div>
-      </div>
-    </div>
-    <%= render 'welcome_box' %>
-  </div>
-</div>
+<%= mount_external_logout %>
```

Comment:
this page is a pure react page, can we use client routing instead of mounting ?

Reply:
It seem that we will need to modify the whole users controller,
I prefer to do it in a separate PR :)